e2e/testcases/managed_resources_test.go

```diff
@@ -22,94 +22,95 @@ import (
-//  * configsync.gke.io/resource-id
+// Drift Correction uses the following metadata:
+//  * The "configmanagement.gke.io/managed" annotation must be set to "enabled".
+//  * The "configsync.gke.io/resource-id" annotation must match the object ID.
+//  * The "configsync.gke.io/manager" annotation must exists and match the RSync's ResourceManager name.
+//  * The "applyset.kubernetes.io/part-of" label must match the RSync's ApplySet ID.
 //
-// The drift correction in the multi-repo mode utilizes the following three annotations:
-//  * configmanagement.gke.io/managed
-//  * configsync.gke.io/resource-id
-//  * configsync.gke.io/manager
-//
-// The drift prevention is only supported in the multi-repo mode, and utilizes the following Config Sync metadata:
-//  * the configmanagement.gke.io/managed annotation
-//  * the configsync.gke.io/resource-id annotation
-//  * the configsync.gke.io/declared-version label
-
-// The reason we have both TestKubectlCreatesManagedNamespaceResourceMonoRepo and
-// TestKubectlCreatesManagedNamespaceResourceMultiRepo is that the mono-repo mode and
-// CSMR handles managed namespaces which are created by other parties differently:
-//   * the mono-repo mode does not remove these namespaces;
-//   * CSMR does remove these namespaces.
-
-// TestKubectlCreatesManagedNamespaceResourceMultiRepo tests the drift correction regarding kubectl
-// tries to create a managed namespace in the the multi-repo mode.
-func TestKubectlCreatesManagedNamespaceResourceMultiRepo(t *testing.T) {
+// Drift Prevention uses the following metadata:
+//  * The "configmanagement.gke.io/managed" annotation must be set to "enabled".
+//  * The "configsync.gke.io/resource-id" annotation must match the object ID.
+
+// TestDriftKubectlApplyClusterScoped tests drift correction after
+// cluster-scoped changes are made with kubectl.
+func TestDriftKubectlApplyClusterScoped(t *testing.T) {
 	nt := nomostest.New(t, nomostesting.DriftControl, ntopts.Unstructured)
 
+	rootSync2Name := "abcdef"
+	rootSync1ApplySetID := applyset.IDFromSync(configsync.RootSyncName, declared.RootScope)
+	rootSync2ApplySetID := applyset.IDFromSync(rootSync2Name, declared.RootScope)
+	rootSync1Manager := declared.ResourceManager(declared.RootScope, configsync.RootSyncName)
+	rootSync2Manager := declared.ResourceManager(declared.RootScope, rootSync2Name)
+
 	namespace := k8sobjects.NamespaceObject("bookstore")
 	nt.Must(nt.RootRepos[configsync.RootSyncName].Add("acme/ns.yaml", namespace))
 	nt.Must(nt.RootRepos[configsync.RootSyncName].CommitAndPush("add a namespace"))
-	if err := nt.WatchForAllSyncs(); err != nil {
-		nt.T.Fatal(err)
-	}
+	nt.Must(nt.WatchForAllSyncs())
 
 	/* A new test */
-	ns := []byte(`
-apiVersion: v1
-kind: Namespace
-metadata:
-  name: test-ns1
-  annotations:
-    configmanagement.gke.io/managed: enabled
-    configsync.gke.io/resource-id: _namespace_test-ns1
-    configsync.gke.io/manager: :root
-`)
+	ns1Obj := &unstructured.Unstructured{}
```

Comment:
nit: can we use `k8sobjects` helpers for these instead of unstructured? It simplifies the setup a bit

Reply:
Probably. But SerializeObject will convert them back to Unstructured after. As long as it handles adding the GVKs from the scheme it should work though.

Reply:
done.